src/libraries/System.Text.RegularExpressions/tools/GenerateRegexCasingTable.csproj

```diff
@@ -2,8 +2,7 @@
 
   <PropertyGroup>
       <OutputType>Exe</OutputType>
-      <TargetFramework>net6.0</TargetFramework>
-      <LangVersion>latest</LangVersion>
+      <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
```

Comment:
Again, please revert this change. This project doesn't import any of the repo's msbuild infrastructure.

Reply:
```suggestion
      <TargetFramework>net7.0</TargetFramework>
```

Reply:
That patch should be removed as the library isn't part of the repo's build.

Reply:
it is removed from installer and we wont be needing any patchs with these changes.